augly/text/augmenters/typo.py

```diff
@@ -104,6 +105,8 @@ def __init__(
             assert (
                 misspelling_dict_path is not None
             ), "'misspelling_dict_path' must be provided if 'typo_type' is 'all' or 'misspelling'"
+            assert max_typo_length >= 1, "Must set 'max_typo_length' >= 1"
+            self.max_typo_length = max_typo_length
```

Comment:
Ah yes, you're only setting `self.max_typo_length` here **if** the `typo_type` passed in is "all" or "misspelling". But we need the below for loop to work even if `typo_type` is "keyboard" or "charmix". So to fix this error please set `self.max_typo_length` to 1 by default if `typo_type` is "keyboard" or "charmix" (so that the for loop only runs once by default).